Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN ./manage.py collectstatic --noinput
 RUN apt-get clean && apt-get purge
 USER mitodl
 
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
Weird! [Here's my successful traceback.](https://gist.github.com/singingwolfboy/c24a0785db48e68911f8c5260efa509d). Can you make sure you're running the latest version of Docker and Docker Compose? I don't know if that would have anything to do with it, but I suppose it's worth checking.